Missing protobufjs/minimal

I got this error when starting with the profiler on App Engine Flex:

module.js:471
throw err;
^
Error: Cannot find module 'protobufjs/minimal'
at Function.Module._resolveFilename (module.js:469:15)
at Function.Module._load (module.js:417:25)
at Module.require (module.js:497:17)
at require (internal/module.js:20:19)
at Object.<anonymous> (/home/nolanmar/src/nolanmartestappflex/nodejs_mvms_quickstart-2017-11-16-16-28/1-hello-world/node_modules/@google-cloud/profiler/proto/profile.js:4:17)
at Module._compile (module.js:570:32)
at Object.Module._extensions..js (module.js:579:10)
at Module.load (module.js:487:32)
at tryModuleLoad (module.js:446:12)
at Function.Module._load (module.js:438:3)

--- a/package.json
+++ b/package.json
@@ -33,6 +33,7 @@
     "nan": "^2.1.0",
     "parse-duration": "^0.1.1",
     "pify": "^3.0.0",
+    "protobufjs": "~6.8.0",
     "request": "^2.83.0",
     "retry-request": "^3.0.1"
   },
@@ -52,7 +53,6 @@
     "mocha": "^4.0.1",
     "nock": "^9.0.22",
     "nyc": "^11.2.1",
-    "protobufjs": "~6.8.0",
     "sinon": "^4.0.1",
     "ts-mockito": "^2.2.5",
     "typescript": "~2.6.x"

